### PR TITLE
Fixes #8134: Do not use proxy when calling localhost in techniques

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8134

### CODE REVIEW

File: initial-promises/node-server/server-roles/1.0/alive-check.cf [hunk 1]
```diff
@@ -54,7 +54,7 @@ bundle agent generic_alive_check(app)
 
     root_server::
 
-      "/usr/bin/curl -s ${site_to_check} |/bin/grep -q OK"
+      "/usr/bin/curl --proxy -s ${site_to_check} |/bin/grep -q OK"
```
Comment (ncharles) [inline initial-promises/node-server/server-roles/1.0/alive-check.cf]: should be --proxy ''
isn't it ?

Comment (matya) [inline initial-promises/node-server/server-roles/1.0/alive-check.cf]: it should.

**Comment by peckpeck** [comment]
PR updated

**Comment by amousset** [comment]
Ok, merging.

**Pull request merged**